src/cipher/encryptingstorage/tests/test_zodb.py

```diff
@@ -476,7 +539,9 @@ def test_suite():
             'encryptingstoragetests.%s' % class_.__name__)
         suite.addTest(s)
 
+    suite.addTest(unittest.makeSuite(TestIterator))
+    suite.addTest(unittest.makeSuite(TestServerEncryptingStorage))
```

Comment:
I would normally recommend `unittest.defaultTestLoader.loadFromName(__name__)`, but ... what on Earth is going on with those test classes up above?  Ouch.